src/Apps/W1/Shopify/App/src/Catalogs/Tables/ShpfyCatalog.Table.al

```diff
@@ -58,69 +63,93 @@ table 30152 "Shpfy Catalog"
             DataClassification = CustomerContent;
             TableRelation = "Gen. Business Posting Group";
+            ToolTip = 'Specifies which Gen. Bus. Posting Group is used to calculate the prices in the catalog.';
         }
         field(9; "VAT Bus. Posting Group"; Code[20])
         {
             Caption = 'VAT Bus. Posting Group';
             DataClassification = CustomerContent;
             TableRelation = "VAT Business Posting Group";
+            ToolTip = 'Specifies which VAT. Bus. Posting Group is used to calculate the prices in the catalog.';
         }
         field(10; "Tax Area Code"; Code[20])
         {
             Caption = 'Tax Area Code';
             DataClassification = CustomerContent;
             TableRelation = "Tax Area";
+            ToolTip = 'Specifies which Tax Area Code is used to calculate the prices in the catalog.';
         }
         field(11; "Tax Liable"; Boolean)
         {
             Caption = 'Tax Liable';
             DataClassification = CustomerContent;
+            ToolTip = 'Specifies if Tax Liable is used to calculate the prices in the catalog.';
         }
         field(12; "VAT Country/Region Code"; Code[10])
         {
             Caption = 'VAT Country/Region Code';
             DataClassification = CustomerContent;
             TableRelation = "Country/Region";
+            ToolTip = 'Specifies which VAT Country/Region Code is used to calculate the prices in the catalog.';
         }
         field(13; "Customer Posting Group"; Code[20])
         {
             Caption = 'Customer Posting Group';
             DataClassification = CustomerContent;
             TableRelation = "Customer Posting Group";
+            ToolTip = 'Specifies which Customer Posting Group is used to calculate the prices in the catalog.';
         }
         field(14; "Prices Including VAT"; Boolean)
         {
             Caption = 'Prices Including VAT';
             DataClassification = CustomerContent;
+            ToolTip = 'Specifies if the prices are Including VAT.';
         }
         field(15; "Allow Line Disc."; Boolean)
         {
             Caption = 'Allow Line Disc.';
             DataClassification = CustomerContent;
+            ToolTip = 'Specifies if line discount is allowed while calculating prices for the catalog.';
         }
         field(16; "Sync Prices"; Boolean)
         {
             Caption = 'Sync Prices';
             DataClassification = CustomerContent;
+            ToolTip = 'Specifies if the prices are synced to Shopify.';
         }
         field(17; "Customer No."; Code[20])
         {
             Caption = 'Customer No.';
             DataClassification = CustomerContent;
             TableRelation = "Customer";
+            ToolTip = 'Specifies the customer''s no.  When Customer No. is Selected: Parameters like ''Customer Discount Group'', ''Customer Price Group'', and ''Allow Line Discount'' on the customer card take precedence over catalog settings';
```

Comment:
Weird capitalizations.
"... customer's no." (this period is the end of sentence) and "When Customer No. ..."